packages/api/lib/granules.js

```diff
@@ -62,10 +66,48 @@ function getGranuleProductVolume(granuleFiles = []) {
     .reduce((x, y) => x + y, 0);
 }
 
+/**
+ * Delete a Granule from Postgres and Dynamo, delete the Granule's
+ * Files from Postgres and S3
+ *
+ * @param {Object} params
+ * @param {Knex} params.knex - DB client
+ * @param {Object} params.dynamoGranule - Granule from DynamoDB
+ * @param {PostgresGranule} params.pgGranule - Granule from Postgres
+ * @param {FilePgModel} params.filePgModel - File Postgres model
+ * @param {GranulePgModel} params.granulePgModel - Granule Postgres model
+ * @param {Object} params.granuleModelClient - Granule Dynamo model
+ */
+const deleteGranuleAndFiles = async ({
+  knex,
+  dynamoGranule,
+  pgGranule,
+  filePgModel = new FilePgModel(),
+  granulePgModel = new GranulePgModel(),
+  granuleModelClient = new Granule(),
```

Comment:
Are you still having an issue with this instantiation not working? If so I'll have to check out the code locally because it looks right to me